integration/init_test.go

```diff
@@ -0,0 +1,85 @@
+package integration_test
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	"github.com/paketo-buildpacks/occam"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/format"
+)
+
+var buildpackInfo struct {
+	Buildpack struct {
+		ID   string
+		Name string
+	}
+	Metadata struct {
+		Dependencies []struct {
+			Version string
+		}
+	}
+}
+
+var buildpacks struct {
+	PhpDist         string `json:"php-dist"`
+	Composer        string `json:"composer"`
+	ComposerInstall string
+	PhpWeb          string `json:"php-web"`
```

Comment:
As mentioned above, PHP Web will be going away soon so may want to test with the newer buildpacks. If they haven't been released yet, you can still test with them by pulling them locally and updating the `integration.json` to include the full path to the buildpack on your local filesystem. And then inside the `integration/init_test.go` you can add `WithVersion("1.2.3").` to that buildpack's `buildpackStore.Get`

Reply:
Cool. I was trying to show that this buildpack (or really `composer`+`composer-install`) fits into the existing chain of PHP buildpacks. I'll update this to the newer buildpacks.